test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -452,7 +459,7 @@ void OnKeyEvent(object source, Window.KeyEventArgs e)
                 }
                 case "f":
                 {
-                    if (mModelAnimation?.State == Animation.States.Playing)
+                    if (mModelAnimation != null && mModelAnimation?.State == Animation.States.Playing)
```

Comment:
```suggestion
                    if (mModelAnimation != null && mModelAnimation.State == Animation.States.Playing)
```
(Minor) 앞에서 null check를 했으니 ?를 빼는것이 좋을것 같습니다~

Reply:
감사합니다. 수정했습니다.